gettsim/social_insurance_contributions/arbeitsl_v.py

```diff
@@ -34,6 +34,83 @@ def sozialv_beitr_gesamt_m(
     return out
 
 
+def sozialv_beitr_arbeitg_m(
+    ges_pflegev_beitr_arbeitg_m: float,
+    ges_krankenv_beitr_arbeitg_m: float,
+    ges_rentenv_beitr_arbeitg_m: float,
+    arbeitsl_v_beitr_arbeitg_m: float,
+) -> float:
+    """Sum of all social insurance contributions of a respective employer.
+
+
+    Parameters
+    ----------
+    ges_pflegev_beitr_arbeitg_m
+        See :func:`ges_pflegev_beitr_arbeitg_m`.
+    ges_krankenv_beitr_arbeitg_m
+        See :func:`ges_krankenv_beitr_arbeitg_m`.
+    ges_rentenv_beitr_arbeitg_m
+        See :func:`ges_rentenv_beitr_arbeitg_m`.
+    arbeitsl_v_beitr_arbeitg_m
+        See :func:`arbeitsl_v_beitr_arbeitg_m`.
+
+    Returns
+    -------
+
+    """
+    out = (
+        ges_pflegev_beitr_arbeitg_m
+        + ges_krankenv_beitr_arbeitg_m
+        + ges_rentenv_beitr_arbeitg_m
+        + arbeitsl_v_beitr_arbeitg_m
+    )
+    return out
+
+
+def sozialv_beitr_gesamt_m(
+    sozialv_beitr_arbeitn_m: float,
+    sozialv_beitr_arbeitg_m: float,
+    in_gleitzone: bool,
+    _ges_pflegev_beitr_midijob_sum_arbeitn_arbeitg_m: float,
+    _arbeitsl_v_beitr_midijob_sum_arbeitn_arbeitg_m: float,
+    _ges_krankenv_midijob_sum_arbeitn_arbeitg_m: float,
+    _ges_rentenv_beitr_midijob_sum_arbeitn_arbeitg_m: float,
+) -> float:
+    """Sum of all social insurance contributions of an employer and employee.
+
+    Parameters
+    ----------
+    sozialv_beitr_arbeitn_m
+        See :func:`sozialv_beitr_arbeitn_m`.
+    sozialv_beitr_arbeitg_m
+        See :func:`sozialv_beitr_arbeitg_m`.
+    _ges_rentenv_beitr_midijob_sum_arbeitn_arbeitg_m
+        See :func:`_ges_rentenv_beitr_midijob_sum_arbeitn_arbeitg_m`.
+    _arbeitsl_v_beitr_midijob_sum_arbeitn_arbeitg_m
+        See :func:`_arbeitsl_v_beitr_midijob_sum_arbeitn_arbeitg_m`.
+    _ges_krankenv_midijob_sum_arbeitn_arbeitg_m
+        See :func:`_ges_krankenv_midijob_sum_arbeitn_arbeitg_m`.
+    _ges_rentenv_beitr_midijob_sum_arbeitn_arbeitg_m
+        See :func:`_ges_rentenv_beitr_midijob_sum_arbeitn_arbeitg_m`.
+    in_gleitzone
+        See :func:`in_gleitzone`.
+
+    Returns
+    -------
+
+    """
+    if in_gleitzone:
```

Comment:
Why is it not always the sum of `sozialv_beitr_arbeitn_m` and `sozialv_beitr_arbeitg_m`?

Reply:
I added this in order to also use and test the functions in `ges_pflegev.py`, `ges_rentenv.py`, `ges_krankenv.py` and `arbeitsl_v.py` that calculate the total social insurance contributions for midijob. Shall I change it so that it is consistent with the other cases? Maybe we will then have some coverage problems again..